packages/input-phone/src/PhoneInput.js

```diff
@@ -206,6 +206,7 @@ PhoneInput.defaultProps = {
   textSelectCountryCancel: "Cancel",
   textSelectCountrySearch: "Search",
   disabled: false,
+  fullScreenBreakpoint: "500px",
```

Comment:
Why 500px? 🤔

Reply:
This value was already set, I just moved from a const to a prop